pkg/secrets/generate_test.go

```diff
@@ -0,0 +1,67 @@
+package secrets
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInflateRanges(t *testing.T) {
+	for _, tt := range []struct {
+		name string
+		want string
+	}{
+		{
+			name: "A-Z",
+			want: "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+		},
```

Comment:
Can we add a test case for empty ranges being passed too? Mainly to make certain we don't have any chance of hitting an out of bounds panic.

Reply:
done